requirements/pytorch/devel.txt

```diff
@@ -4,11 +4,5 @@
 # install all extra dependencies for full package testing
 -r ./extra.txt
 
-# install all loggers for full package testing
```

Comment:
can we rather have another file ci.txt with these simplified options?

Reply:
What would you call it?

Reply:
And in any case, why have it if we don't use it?

And for users, why would any user want to install all loggers?